packages/graphql/src/schema/microLearning.ts

```diff
@@ -37,5 +39,7 @@ export const MicroLearning = MicroLearningRef.implement({
       type: [ElementStackRef],
       nullable: true,
     }),
+
+    numOfInstances: t.exposeInt('numOfInstances', { nullable: true }),
```

Comment:
will be replace through numOfStacks (which is much more accurate) through the next PR